CHANGELOG.md

```diff
@@ -20,15 +20,19 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 - server_security_misconfiguration.no_rate_limiting_on_form.sms_triggering
 - server_security_misconfiguration.mail_server_misconfiguration.email_spoofing_on_email_domain
 - server_security_misconfiguration.insecure_ssl.certificate_error
+- cross_site_scripting_xss.stored.privileged_user_to_privilege_elevation
+- cross_site_scripting_xss.stored.privileged_user_to_no_privilege_elevation
 
 ### Removed
 - server_security_misconfiguration.mail_server_misconfiguration.missing_spf_on_email_domain
 - server_security_misconfiguration.mail_server_misconfiguration.email_spoofable_via_third_party_api_misconfiguration
+- cross_site_scripting_xss.stored.admin_to_anyone
 
 ### Changed
 - broken_authentication_and_session_management.failure_to_invalidate_session.on_password_change updated remediation advice
 - CWE mapping default changed from `[CWE-2000]` to `null`
 - Updated python version to 3.6
+- cross_site_scripting_xss.stored.non_admin_to_anyone name changed from "Non-Admin to Anyone" to "Non-Privileged User to Anyone"
```

Comment:
we don't want to change the id as well?

Reply:
I didn't want to deprecate this ID, but we sure can if we should update it.